poll.js

```diff
@@ -0,0 +1,150 @@
+'use strict'
+var imgs = ['bag.jpg',
+            'banana.jpg',
+            'bathroom.jpg',
+            'boots.jpg',
+            'breakfast.jpg',
+            'bubblegum.jpg',
+            'chair.jpg',
+            'cthulhu.jpg',
+            'dog-duck.jpg',
+            'dragon.jpg',
+            'pen.jpg',
+            'pet-sweep.jpg',
+            'scissors.jpg',
+            'shark.jpg',
+            'sweep.png',
+            'tauntaun.jpg',
+            'unicorn.jpg',
+            'usb.gif',
+            'water-can.jpg',
+            'wine-glass.jpg'];
+
+var choices = [];
+choices.push(document.getElementById('picture0'));
+choices.push(document.getElementById('picture1'));
+choices.push(document.getElementById('picture2'));
+var results = document.getElementById('results');
+var rlist = document.getElementById('resultslist');
+
+/**
+ * [Image description]
+ * @param {[type]} file [description]
+ */
+function Image(file) {
+  this.score = 0; //times selected
+  this.appeared = 0; //times appeared on the screen
```

Comment:
Can you think of a variable name that's clear enough that you don't have to write a comment explaining what it is right afterwards?